datafusion/sql/tests/integration_test.rs

```diff
@@ -3112,6 +3112,15 @@ fn test_prepare_statement_to_plan_panic_param_format() {
     logical_plan(sql).unwrap();
```

Comment:
We could probably update this previous test to change Internal / Plan rather than adding a new test

Reply:
Sure, no problem. This test is not needed then.